frontend/ContextMenu.js

```diff
@@ -58,16 +63,39 @@ class ContextMenu extends React.Component {
     this.props.hideContextMenu();
   }
 
+  getHeight = element => {
```

Comment:
I'm honestly not sure how those even got added to this branch.  I didn't realize they were there until I made the pull request.